src/lib/useEthereumTransaction.js

```diff
@@ -119,13 +128,15 @@ export default function useEthereumTransaction(
             chainId,
             gasPrice: gasPrice.toFixed(0),
             gasLimit,
+            txnSigner,
           })
         )
       );
 
       setSignedTransactions(txns);
       setState(STATE.SIGNED);
     } catch (error) {
+      console.error(error);
```

Comment:
```suggestion
      console.error('error signing txs', error);
```

We might not always be able to see a legible trace of where the error came from, so labeling it seems nice.